src/core/serpents.erl

```diff
@@ -51,13 +54,14 @@ start_phase(start_cowboy_listeners, _StartType, []) ->
     , cowboy_swagger_handler
     ],
   Trails =
-    [ {"/", cowboy_static, {file, "www/index.html"}}
-    , {"/favicon.ico", cowboy_static, {file, "www/assets/favicon.ico"}}
-    , {"/assets/[...]", cowboy_static, {dir, "www/assets"}}
-    , {"/game/:game_id", cowboy_static, {file, "www/game.html"}}
+    [ trails:trail("/", cowboy_static, {file, "www/index.html"})
+    , trails:trail(
+        "/favicon.ico", cowboy_static, {file, "www/assets/favicon.ico"})
+    , trails:trail("/assets/[...]", cowboy_static, {dir, "www/assets"})
+    , trails:trail("/game/:game_id", cowboy_static, {file, "www/game.html"})
     | trails:trails(Handlers)
     ],
-  trails:store(Trails),
+  ok = trails:store(Trails),
```

Comment:
@HernanRivasAcosta This warning is only here because @gadgetci still runs Erlang/OTP 17.x … in 18 is fixed.